src/components/message/text-to-emojis.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Emoji } from 'emoji-mart';
+import nodeEmoji from 'node-emoji';
+
+// eslint-disable-next-line no-useless-escape
+const emojiRegex = new RegExp('(:[a-zA-Z0-9-_+]+:(:skin-tone-[2-6]:)?)', 'g');
+
+const fallback = (text) => () => text;
+
+export const textToEmojis = (message: string) => {
```

Comment:
I'm not entirely sure we need this any more since we actually save the utf character and not the colon string.

Reply:
when I remove it we render emoji as plain text

Reply:
Maybe on legacy messages where we saved the colon token but if we've saved the utf character then plain text is exactly what it is.

Reply:
hm, do you have an idea why we still have `textToEmojis` in zero-web?

Reply:
I believe we left it in to smooth the transition between messages that had the colon token and the new utf messages. But, it's been 2 months now so we shouldn't need it any more.

Having said that, if you did try it and it still doesn't render the utf characters correctly, then just leave it in.

Reply:
you were right about this, I've deleted it.